slothy/core/config.py

```diff
@@ -1111,6 +1141,11 @@ def __init__(self, Arch, Target):
         self._selfcheck_failure_logfile = None
         self._allow_useless_instructions = False
 
+        # TODO: This should be False by default, but this is a breaking
+        # change that requires a lot of examples (where it _is_ safe to
+        # apply address offset fixup) to be changed.
+        self._unsafe_address_offset_fixup = True
```

Comment:
Do we want to track this in an issue or is this going to be  around forever?

Reply:
Can you open an issue?

Reply:
done : https://github.com/slothy-optimizer/slothy/issues/101